anyway/infographics_utils.py

```diff
@@ -475,16 +475,18 @@ def get_accident_count_by_accident_type(location_info, start_time, end_time):
         return merged_accident_type_count
 
 
+# just for pybabel
+_("Fatal and severe accidents heat map")
+
+
```

Comment:
Seems like a duplicate of line 518 - `_(AccidentsHeatMapWidget.BASE_TITLE)`

Reply:
The reason for line 479 is I couldn't make `pybabel` extract the string when `_()` is called on a variable (line 518).

Reply:
Try to use `_()` on line 485 instead

Reply:
@ziv17 correct me if I'm wrong - I believe that these strings that are localized in the file and not in the localize_items functions - aren't translated when data is a called from cache.
For example - see the widget `head_on_collisions_comparison`: specific_road_segment_fatal_accidents is not translated to Hebrew (comparing [this call](https://www.anyway.co.il/api/infographics-data?news_flash_id=16577&years_ago=5&lang=he) and [this call](https://www.anyway.co.il/api/infographics-data?news_flash_id=16577&years_ago=5&lang=en)) - the string specific_road_segment_fatal_accidents is not localized. Did you mean to localize it?

Reply:
> 
> 
> Try to use `_()` on line 485 instead

I did not think it will work, but it does work. When the file is loaded, and `lang` is not set it returns the input.

Reply:
> 
> 
> @ziv17 correct me if I'm wrong - I believe that these strings that are localized in the file and not in the localize_items functions - aren't translated when data is a called from cache.
> For example - see the widget `head_on_collisions_comparison`: specific_road_segment_fatal_accidents is not translated to Hebrew (comparing [this call](https://www.anyway.co.il/api/infographics-data?news_flash_id=16577&years_ago=5&lang=he) and [this call](https://www.anyway.co.il/api/infographics-data?news_flash_id=16577&years_ago=5&lang=en)) - the string `specific_road_segment_fatal_accidents` is not localized. Did you mean to localize it?

Here it is localized on purpose. `gettext()` is called twice, once when the constant is set, and then when the constant is used in `localize_items()`. Just to help the `pybabel extract`

Reply:
Not sure I understand but I think it's better not no localize `specific_road_segment_fatal_accidents` in `head_on_collisions_comparison` Widget at the moment - since at the moment the FE localized it there - we'll change that in the coming future.